src/WordPress.php

```diff
@@ -845,10 +882,26 @@ private function create_site( $assoc_args ) {
 		}
 
 		if ( ! empty( $this->cache_type ) ) {
+
 			$this->enable_object_cache();
 			$this->enable_page_cache();
+
+			if ( $this->is_vip ) {
+				EE::warning( 'Nginx-helper and wp-redis plugin is installed to enable cache. Please add it in your .gitignore to avoid it from git diff and commit' );
+				EE::log( 'Note: Redis cache is setup for this site so it will use wp-redis object cache but not the memcache which is mentioned in VIP development doc from mu-plugin drop-ins.' );
+			}
+		}
+
+		$wp_root_dir = $this->site_data['site_fs_path'] . '/app/htdocs';
+
+		if ( $this->is_vip && file_exists( $wp_root_dir . '/mu-plugins' ) ) {
+			// Enable VIP MU plugins.
+			EE::exec( 'mv ' . $wp_root_dir . '/mu-plugins ' . $wp_root_dir . '/wp-content/' );
 		}
 
+		// Reset wp-content permission which may have been changed during git clone from host machine.
+		EE::exec( "docker-compose exec --user=root php chown -R www-data: /var/www/htdocs/wp-content" );
+
```

Comment:
One thing we need to cover is if existing repo has mu-plugin we should rename it to client-mu-plugin or atleast output a message/warning for same. @rahulsprajapati